specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml

```diff
@@ -1,7 +1,11 @@
 emit:
   - "@azure-tools/typespec-autorest"
-  - "@azure-tools/cadl-apiview"
-# options:
+#  - "@azure-tools/typespec-apiview"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "cadl-output"
+    emitter-output-dir: "{project-root}"
+    output-file: "{azure-resource-provider-folder}/{version}/openapi.json"
```

Comment:
Should this output to this folder instead?

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane/AzureOpenAI/authoring

If so, I think this whole directory should be deleted:

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/OpenAI.Authoring/cadl-output

Reply:
It is up to you - the swagger here does not (and never did) match the swagger in the associated data-plane directory.  We could emit it to a different folder in data-plane, but I feel like having the swagger in this directory or removing it altogether are the least bad options.

Reply:
I'm confused.  If the swagger generated from TypeSpec does not match the swagger in the data-plane directory, why is it being generated at all?  Why does this TypeSpec folder exist in the repo at all?

Reply:
I am guessing that the swagger was there before and this is a proof of concept.  There are multiple versions in the data-plane folder, and this spec does not even represent the latest one.

Reply:
OK, I'm fine merging this PR as-is with one additional change to remove this file I believe is an extra copy that shouldn't be there:

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Authoring/cadl-output/openapi.json

Reply:
Later we will need to decide what to do with specs that are proof-of-concept or client-only (don't emit swagger).

Reply:
If this is a proof of concept I would argue we should have it merged into the main branch. If we plan to ship this we should have it outputting specs where they are supposed to be going and should definitely not have the name "cadl-output" in the path.

Reply:
@lmazuel: Do you know the history of the `OpenAI.Authoring` spec, and who is maintaining it going forward?

Reply:
Let me ask on our internal chat